COCOAPODS.md

```diff
@@ -1,150 +1,343 @@
     end
     ```
+    
+    </div>
 
+4. Re-import the project.
+
+### Add a dependency between a Kotlin Pod with an Xcode project with several targets
+
+1. Create an Xcode project with `Podfile` if you haven’t done this yet.
+2. Add the path to your `Podfile` with `podfile = projectDir.resolve("..")` to `build.gradle.kts` (`build.gradle`) of
+ your Kotlin project.
+3. Add dependencies to Pod libraries that you want to use in your project with `pod()`.
+4. For each target, specify the minimum target version for the Pod library.
+
+    <div class="sample" markdown="1" theme="idea" data-highlight-only>
+    
+    ```kotlin
+    kotlin {
+        ios()
+        tvos()
+    
+        cocoapods {
+            summary = "CocoaPods test library"
+            homepage = "https://github.com/JetBrains/kotlin"
+            ios.deploymentTarget = "13.5"
+            tvos.deploymentTarget = "13.4"
+       
+            pod("AFNetworking", "~> 4.0.0")
+            podfile = projectDir.resolve("../severalTargetsXcodeProject/Podfile") // specify the path to Podfile
+        }
+    }
+    ```
+    
     </div>
 
-4. Run `pod install` for the Xcode project.
+2. Add the name and path of the Kotlin Pod you want to include in the Xcode project to `Podfile`.
+
+    <div class="sample" markdown="1" theme="idea" mode="ruby" data-highlight-only>
+    
+    ```ruby
+    target 'iosApp' do
+      use_frameworks!
+      platform :ios, '13.5'
+      # Pods for iosApp
+      pod 'kotlin_library', :path => '../kotlin-library'
+    end
     
-After completing these steps, you can
-open the generated workspace (see [CocoaPods documentation](https://guides.cocoapods.org/using/using-cocoapods.html#installation))
-and run an Xcode build.
+    target 'TVosApp' do
+      use_frameworks!
+      platform :tvos, '13.4'
+      
+      # Pods for TVosApp
+      pod 'kotlin_library', :path => '../kotlin-library'
+    end
+    ```
+    
+    </div>
+    
+3. Re-import the project.
 
-## Interoperability
+You can find a sample project [here](https://github.com/zoldater/severalXcodeTargetsDemo).
 
-The CocoaPods plugin also allows using CocoaPods libraries without manual configuring cinterop
-parameters (see the [corresponding section](https://kotlinlang.org/docs/reference/building-mpp-with-gradle.html#cinterop-support)
-of the multiplatform plugin documentation). The `cocoapods { ... }` code block allows you to
-add dependencies on CocoaPods libraries.
+### Add dependencies between multiple Kotlin Pods and an Xcode project
 
-<div class="sample" markdown="1" theme="idea" mode="kotlin">
+1. Create an Xcode project with `Podfile` if you haven’t done this yet.
+2. Create a hierarchical structure of your Kotlin projects - a root and child projects.
+3. Add the path to your `Podfile` with `podfile = projectDir.resolve("..")` to `build.gradle.kts` (`build.gradle`) of
+ the root project.
+4. Specify that you don’t need a `Podspec` for this root project - `noPodspec()`.
```

Comment:
This is only necessary if root project is not represent Kotlin Pod

Reply:
Added this info.